community.py

```diff
@@ -37,13 +37,11 @@ def post(self):
 		return hsh_to_return
 
 class AddToCommunity(Resource):
-	def post(self):
+	def post(self,user_id):
 		# Inputs
-		API_data = request.get_json()
 		API_KEY = request.headers['THE_API_KEY']
```

Comment:
Are we validating the api key right now? If all endpoints will require an API key is there middlewear we can include that validates the key before reaching the logic of the endpoint? If many endpoints require the api key this seems like we'll be writing duplicate logic for each endpoint.

Reply:
I agree we should do that, 
Below are the code changes that we need to make.
- Interface with the db using an ORM (as discussed on slack).
- Create a global method to validate API-KEY.

Reply:
Alright that sounds good - it's okay if we address this in a follow up pr since this seems like a feature in itself.